src/sources/MFEMScalarPotentialSource.C

```diff
@@ -9,6 +9,8 @@ MFEMScalarPotentialSource::validParams()
   params.addRequiredParam<std::string>(
       "potential",
       "Name of the potential used in the solver, necessary to find boundary conditions");
+  params.addRequiredParam<std::string>("grad_electric_potential",
```

Comment:
`MFEMScalarPotentialSource` should be agnostic as to whether it's solving for an electric or magnetic potential - probably worth renaming this to just `"grad_potential"`.